docs/data_node/extract_node/oracle-cdc.md

```diff
@@ -328,6 +328,13 @@ TODO: It will be supported in the future.
       <td>String</td>
       <td>Inlong metric label, format of value is groupId&streamId&nodeId.</td> 
     </tr>
+    <tr>
+       <td>source.multiple.enable</td>
+       <td>optional</td>
+       <td style={{wordWrap: 'break-word'}}>false</td>
+       <td>Boolean</td>
+       <td>Whether to enable multiple schema and table migration. If it is' true ', Oracle Extract Node will compress the physical field of the table into a special meta field' data 'in the format of 'canal json'.</td> 
```

Comment:
data -> data_canal or data_debezium, data is deprecated

Reply:
done.